### PR TITLE
Feature/centralize styles

### PR DESCRIPTION
**Pull request by ghost**
Adds style using emotion's `Global` to all 2019 packages and removes `global.styles.css`. It also updates the template

### CODE REVIEW

**Review by jaronheard — APPROVED** [review]
Looks great!

Just something I was thinking about while reviewing, it'd be great to have our template using the Emotion 10 syntax to show good example code

```
/** @jsx jsx */ 
import { jsx, css } from "@emotion/core"

/* css prop instead of className prop */
```

File: _templates/package-generator/with-prompt/src/components/App/index.ejs.t [hunk 1]
```diff
@@ -36,44 +35,59 @@ const paragraphStyle = css`
...
+  <Fragment>
     <Global styles={BrandTheme} />
     <PageLayout
       teamTitle="<%=title%>"
       heroTitle="🚧 Site Under Construction 🚜"
       heroSubtitle="Playground for building out the <%=year%> <%=title%> team frontend"
       overlay
     >
-      <section className={cx(sectionBodyHeading, sectionMaxWidthSmall)}>
+      <section css={css`
```
Comment (jaronheard) [inline _templates/package-generator/with-prompt/src/components/App/index.ejs.t]: There's actually a nicer way to combine these styles:
`css={[sectionBodyHeading, sectionMaxWidthSmall]}`

[(described here, bottom code block)](https://emotion.sh/docs/composition)

**Review by jaronheard — COMMENTED** [review]
Just one comment about a slightly cleaner way to compose styles using the `css` prop

File: _templates/package-generator/with-prompt/src/components/App/index.ejs.t (hunk 1 above)
Comment (Dianna) [reply _templates/package-generator/with-prompt/src/components/App/index.ejs.t]: Crikey. I just didn't scroll far enough down. Fixing.

**Review by jaronheard — APPROVED** [review]

**Pull request merged**